lib/winston/logger.js

```diff
@@ -231,7 +217,8 @@ class Logger extends Transform {
       const tokens = msg && msg.match && msg.match(formatRegExp);
 
       if (!tokens) {
-        const info = Object.assign({}, this.defaultMeta, meta, {
+        const infoObj = { message: msg };
+        const info = Object.assign({}, this._addDefaultMeta(infoObj), meta, {
```

Comment:
I forget, do we need `{}` in this case or can we just have `this._addDefaultMeta(infoObj)` as the first param now?

Reply:
Will have to retest - I believe it was necessary but I could have just been staying consistent with what was there previously.